City-Bureau/city-scrapers-bismarck
# 🏗️ Build mixin + spiders: bisnd_mc

## What's this PR do?
Adds a mixin and spiders to scrape multiple commission/boards from the website of the City of Mandan in Mandan, ND. The mixin that handles the core scraping logic is called `bisnd_mc`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spiders:
```
scrapy list | grep 'bisnd_mc' | xargs -I {} scrapy crawl {} -O {}.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect the CSVs to ensure the data looks valid. You can view the value in the "source" column of the rows to ensure the data matches.

## Are there any smells or added technical debt to note?


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206618632217643
